docs: document broken upgrade version for tasklist (#5858)
camunda/camunda-docs: versioned_docs/version-8.4/self-managed/operational-guides/update-guide/830-to-840.md

- Zeebe, Operate, and Tasklist now support Amazon OpenSearch `2.5.x`. Note that using Amazon OpenSearch requires setting up a new Camunda installation. A migration from previous versions or Elasticsearch environments is currently not supported.

Please see [supported environments](../../../reference/supported-environments.md) for more a full overview of Camunda environment requirements.

## Tasklist

:::caution

> There is a [bug](https://github.com/camunda/camunda/issues/32208) in Tasklist 8.4.20 which causes Tasklist's importer to fail importing records for the `flownode-instance` index. As such, **you should skip 8.4.20 and update directly to 8.4.21**

Reviewer: ```suggestion
**You should skip 8.4.20 and update directly to 8.4.21** due to a [bug](https://github.com/camunda/camunda/issues/32208) in Tasklist 8.4.20 that causes Tasklist's importer to fail importing records for the `flownode-instance` index.
```
